prody/proteins/psiblast.py

```diff
@@ -29,19 +29,29 @@ def psiBlastRun(sequence='runexample', cycles=2, filename=None, **kwargs):
         default is 2
     :type cycles: int
     """
-    jobId = kwargs.get('jobId',None)
+    psithr = kwargs.get('psithr', 1.0e-3)
+    jobId = kwargs.get('previousjobid','')
+    selectedHits = kwargs.get('selectedHits','selected_hits.txt')
```

Comment:
Try to keep the variable/argument names consistent. While the preferences of different languages are very different, [Python prefers to separate the words by underscores for variables and functions (but not for classes).](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)

Reply:
PEP8 is good Python style guide but surely it is not strict rules.

Reply:
In this case, I am mostly using what EBI PSI-BLAST has. Generally ok.

Reply:
ProDy style is to use lowercase with uppercase for new words in functions. We can have underscores in variables though.

Reply:
I think blastpdb.py and psiblast.py now have this to the extent possible.